src/panels/lovelace/editor/unused-entities/hui-unused-entities.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  PropertyValues,
+  property,
+  customElement,
+  css,
+  CSSResult,
+} from "lit-element";
+
+import { classMap } from "lit-html/directives/class-map";
+import "../../../../components/ha-fab";
+
+import "./hui-select-row";
+
+import { computeRTL } from "../../../../common/util/compute_rtl";
+import { computeUnusedEntities } from "../../common/compute-unused-entities";
+import { navigate } from "../../../../common/navigate";
+
+import { showSelectViewDialog } from "../select-view/show-select-view-dialog";
+import { showEditCardDialog } from "../card-editor/show-edit-card-dialog";
+
+import { HomeAssistant } from "../../../../types";
+import { Lovelace } from "../../types";
+import { LovelaceConfig } from "../../../../data/lovelace";
+
+@customElement("hui-unused-entities")
+export class HuiUnusedEntities extends LitElement {
+  @property() public lovelace?: Lovelace;
+
+  @property() public hass?: HomeAssistant;
+
+  @property() private _unusedEntities: string[] = [];
+
+  private _selectedEntities: string[] = [];
+
+  private get _config(): LovelaceConfig {
+    return this.lovelace!.config;
+  }
+
+  protected updated(changedProperties: PropertyValues): void {
+    super.updated(changedProperties);
+
+    if (changedProperties.has("lovelace")) {
+      if (
+        this.lovelace!.mode === "storage" &&
+        this.lovelace!.editMode === false
+      ) {
+        navigate(this, "/lovelace");
```

Comment:
Is this something this element should deal with ? Shouldn't the parent re-render ?